packages/cli/src/lib/pg/backups.ts

```diff
@@ -49,53 +54,147 @@ function prefix(transfer: BackupTransfer) {
   }
 }
 
-function filesize(size: number, opts = {}): string {
-  Object.assign(opts, {
-    decimalPlaces: 2,
-    fixedDecimals: true,
-  })
-  return bytes(size, opts)
-}
+class Backups {
+  public transfer: Pick<Backups, 'status' | 'num' | 'name'> // legacy purposes only
+  protected app: string
+  protected heroku: APIClient
+  protected logsAlreadyShown = new Set<string>()
```

Comment:
Hmm.. good question. I can't see where this would be required right off hand. Since these instances are created using a factory function and those instances are usually short lived, I don't think explicitly clearing them would be necessary.

Reply:
👍 yeah that was my assumption as well. just wanted to make sure you agreed.